### ISSUE TITLE
Cannot create NPC actors when "Sanity Ability Score" or "Honor Ability Score" option(s) are enabled

### ISSUE THREAD

**Issue by TheGiddyLimit** [foundryvtt/dnd5e#2333]
Env: 

- Foundry v11 300
- dnd5e v2.2.1

---

Reproduction steps:

1) Enable one of the optional ability scores

2)

```js
Actor.create({name: "Test NPC", type: "npc"})
```

![image](https://github.com/foundryvtt/dnd5e/assets/25264887/89764fe4-b6fe-4426-9811-770eba528c2c)

**Comment by arbron** [comment]
Looks like it isn't properly accounting for `existing` possible being undefined. It should read:
```javascript
if ( typeof defaultValue === "string" ) defaultValue = existing?.[defaultValue]?.value ?? initial.value;
```

**Comment by kaelad02** [comment]
I hate to be "that guy" about it, but someone comes into the dnd5e channel Discord and asks about this at least once a day. Would it be possible to make a patch to fix this now instead of waiting for v2.3.0?
